metagraph-plugin/metagraph_katana/plugins/metagraph_katana/algorithms.py

```diff
@@ -39,8 +39,23 @@ def kg_bfs_iter(graph: KatanaGraph, source_node: NodeID, depth_limit: int) -> Nu
         bfs(graph.value, start_node, bfs_prop_name)
     pg_bfs_list = graph.value.get_node_property(bfs_prop_name).to_pandas().values.tolist()
     new_list = [[i, pg_bfs_list[i]] for i in range(len(pg_bfs_list)) if pg_bfs_list[i] < depth_limit_internal]
-    sorted_list = sorted(new_list, key=lambda each: (each[1], each[0]))
-    bfs_arr = np.array([each[0] for each in sorted_list])
+    sorted_list = [start_node]
+
+    new_list.remove([start_node, start_node])
+    
+    i = 0
```

Comment:
how is `i ` being used here?

Reply:
i is used to iterate through sorted_list for every iteration of the while loop.

Reply:
oh sorry